app/src/main/java/com/sun/moneymanagement/utils/SafeClickListener.kt

```diff
@@ -0,0 +1,18 @@
+package com.sun.moneymanagement.utils
+
+import android.os.SystemClock
+import android.view.View
+
+class SafeClickListener(
+        private var defaultInterval: Int = 1000,
+        private val onSafeCLick: (View) -> Unit
+) : View.OnClickListener {
+    private var lastTimeClicked: Long = 0
+    override fun onClick(v: View) {
+        if (SystemClock.elapsedRealtime() - lastTimeClicked < defaultInterval) {
```

Comment:
Nên viết thế này để giúp dễ hiểu.
```suggestion
        val isNotSafeClick = SystemClock.elapsedRealtime() - lastTimeClicked < defaultInterval
        if (isNotSafeClick) {
```